client/src/components/AdminMessageOptions.js

```diff
@@ -0,0 +1,51 @@
+export default function NavBar() {
+  return (
+    <div className="flex ml-auto">
+      <svg
```

Comment:
I would like to suggest Font Awesome Icons for this one sir. Font Awesome Icon uses components directly
REF: https://fontawesome.com/v5/docs/web/use-with/react
Icons: https://fontawesome.com/search
SC: 
![image](https://user-images.githubusercontent.com/109492180/193002039-04072280-f801-4301-a253-13432dbcba44.png)